### PR TITLE
[PWN-7915] Fix empty state on Home screen. Fetch balances when it is …

### PR DESCRIPTION
**Pull request by lisemyon**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-7915

…needed

## Description of the changes
-

### CODE REVIEW

File: p2p_wallet/Extensions/Wallet+Extensions.swift [hunk 1]
```diff
@@ -12,8 +12,12 @@ extension Array where Element == Wallet {
     var totalAmountInCurrentFiat: Double {
         reduce(0) { $0 + $1.amountInCurrentFiat }
     }
-    
+
+    var totalAmount: Double? {
```
Comment (TrGiLong) [inline p2p_wallet/Extensions/Wallet+Extensions.swift:16]: Bad code. There are difference tokens and we can not performance additional operator.

File: p2p_wallet/Extensions/Wallet+Extensions.swift [hunk 2]
```diff
@@ -12,8 +12,12 @@ extension Array where Element == Wallet {
...
+        reduce(0) { $0 + $1.amount }
+    }
+
     var isTotalBalanceEmpty: Bool {
-        totalAmountInCurrentFiat == 0
```
Comment (TrGiLong) [inline p2p_wallet/Extensions/Wallet+Extensions.swift:17]: Use `contains { $0.amountInLamport > 0 }` or `allSatisfy { $0.amountInLamport == 0 }`

File: p2p_wallet/Extensions/Wallet+Extensions.swift (hunk 1 above)
Comment (lisemyon) [reply p2p_wallet/Extensions/Wallet+Extensions.swift:16]: what do you mean? that's just for the empty state. do you think I should hide this param to "isTotalAmountEmpty" or just perform this count in HomeViewModel?

Comment (lisemyon) [reply p2p_wallet/Extensions/Wallet+Extensions.swift:16]: I guess second one is better

**Review by TrGiLong — APPROVED** [review]

**Pull request merged**